modules/drivers/rslidar/rslidar_pointcloud/data/lidar1/curves.csv

```diff
@@ -307,7 +307,7 @@
 5.1331,4.2384,4.6339,3.9479,3.6514,5.667,3.9687,3.6054,3.6245,4.3939,6.0772,2.8687,3.3689,5.2192,4.2651,7.2788
 5.1314,4.2258,4.6339,3.947,3.6529,5.6541,3.9689,3.6054,3.6278,4.3863,6.0737,2.8704,3.3616,5.2157,4.2625,7.2421
 5.1298,4.2136,4.6339,3.9461,3.6544,5.6414,3.9692,3.6054,3.6311,4.3787,6.0701,2.8721,3.3542,5.2124,4.26,7.2057
-5.1282,4.2017,4.6339,3.9452,3.656,5.6288,3.9694,3.6052,3.6345,4.3711,6.0664,2.8739,3.3467,5.2091,4.2575,7.1698
```

Comment:
Are you sure to replace the value of curves here ?

Reply:
Yes. It does not influence the lidar. Each lidar has its own calibration file, so this file will be replaced when users bring up their own lidar.